### PR TITLE
Pal 605 fix data service

### PR DESCRIPTION
**Pull request by nw1984**
Fix for resource service not being present and mount point

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -133,7 +133,8 @@ spec:
             container('maven') {
                 configFileProvider([configFile(fileId: "${env.CONFIG_FILE}", variable: 'MAVEN_SETTINGS')]) {
                     if (("${env.BRANCH_NAME}" == "develop") ||
-                            ("${env.BRANCH_NAME}" == "master")) {
+                            ("${env.BRANCH_NAME}" == "master") ||
+                                    ("${env.BRANCH_NAME}" == "PAL-605-fix-data-service")) {
```
Comment (developer01189998819991197253) [inline Jenkinsfile]: after approved, please revert this back to original

**Review by developer01189998819991197253 — DISMISSED** [review]

File: data-service/mvn_dependency_tree.txt [hunk 2]
```diff
@@ -1,4 +1,5 @@
 uk.gov.gchq.palisade:data-service:jar:0.4.0-SNAPSHOT
++- com.google.code.gson:gson:jar:2.8.5:compile
```
Comment (ac74475) [inline data-service/mvn_dependency_tree.txt:2]: what was the reason for changing the gson version back a bug fix? (2.8.6 -> 2.8.5)

Comment (developer01189998819991197253) [reply data-service/mvn_dependency_tree.txt:2]: 2.8.6 doesnt seem to fix the issues, yet 2.8.5 does!

**Review by ac74475 — DISMISSED** [review]

File: data-service/mvn_dependency_tree.txt (hunk 2 above)
Comment (nw1984) [reply data-service/mvn_dependency_tree.txt:2]: Hi ac74475, the shaded hadoop client has a dependency on gson version <= 2.8.5 - there is an interface: com.google.gson.GsonBuilder com.google.gson.GsonBuilder.setLenient() that has been deleted from 2.8.6 onwards. The gson construction requires this hence the reason that we have to include the specific version.

**Review by ac74475 — APPROVED** [review]

**Review by developer01189998819991197253 — APPROVED** [review]

**Pull request merged**